dotnet/notebooks/07-DALL-E-2.ipynb

```diff
@@ -33,18 +33,18 @@
    "source": [
     "// Usual setup: importing Semantic Kernel SDK and SkiaSharp, used to display images inline.\n",
     "\n",
-    "#r \"nuget: Microsoft.SemanticKernel, 1.0.0-beta6\"\n",
+    "#r \"nuget: Microsoft.SemanticKernel, 1.0.0-rc3\"\n",
     "#r \"nuget: SkiaSharp, 2.88.3\"\n",
     "\n",
     "#!import config/Settings.cs\n",
     "#!import config/Utils.cs\n",
     "#!import config/SkiaUtils.cs\n",
     "\n",
     "using Microsoft.SemanticKernel;\n",
-    "using Microsoft.SemanticKernel.AI.ImageGeneration; \n",
+    "using Microsoft.SemanticKernel.AI.TextToImage;\n",
     "using Microsoft.SemanticKernel.AI.Embeddings;\n",
-    "using Microsoft.SemanticKernel.AI.Embeddings.VectorOperations;\n",
-    "using Microsoft.SemanticKernel.Connectors.AI.OpenAI;"
+    "using Microsoft.SemanticKernel.Connectors.AI.OpenAI;\n",
```

Comment:
Revisit the namespaces.

All `AI` is gone

```diff
- "using Microsoft.SemanticKernel.AI.ImageGeneration; \n",
- "using Microsoft.SemanticKernel.AI.TextToImage;\n",
- "using Microsoft.SemanticKernel.AI.Embeddings;\n",
- "using Microsoft.SemanticKernel.AI.Embeddings.VectorOperations;\n",
- "using Microsoft.SemanticKernel.Connectors.AI.OpenAI;\n"
+ "using Microsoft.SemanticKernel.ImageGeneration; \n",
+ "using Microsoft.SemanticKernel.TextToImage;\n",
+ "using Microsoft.SemanticKernel.Embeddings;\n",
+ "using Microsoft.SemanticKernel.Embeddings.VectorOperations;\n",
+ "using Microsoft.SemanticKernel.Connectors.OpenAI;\n"

Reply:
Thanks. I think namespace update will be available with RC4, which is not released yet :) As soon as RC4 is available, I will update notebooks to use RC4 and update all the namespaces.